v2.1/admin-ui-hardware-dashboard.md

```diff
@@ -8,29 +8,13 @@ The **Hardware** dashboard lets you monitor CPU usage, disk throughput, network
 
 The **Hardware** dashboard displays the following time series graphs:
 
-## User CPU Percent
+## CPU Percent
 
-<img src="{{ 'images/v2.1/admin_ui_user_cpu.png' | relative_url }}" alt="CockroachDB Admin UI User CPU Percent graph" style="border:1px solid #eee;max-width:100%" />
+<img src="{{ 'images/v2.1/admin_ui_cpu_percent.png' | relative_url }}" alt="CockroachDB Admin UI CPU Percent graph" style="border:1px solid #eee;max-width:100%" />
 
-- In the node view, the graph shows the amount of CPU resources used by the CockroachDB process for the selected node.
+- In the node view, the graph shows the amount of CPU resources used by the CockroachDB process as well as the CockroachDB system-level operations for the selected node.
```

Comment:
Also "amount of CPU resources" is a bit weird, since we're only showing one resource: time. How about "…shows the CPU utilization of the CockroachDB process, where 100% means that CockroachDB is fully using all CPUs"

Especially important to explicitly state what 100% means, since people might think it means fully using only one core.